uber/models.py

```diff
@@ -856,6 +861,7 @@ def model_mixin(cls, model):
 
 
 class Group(MagModel, TakesPaymentMixin):
+    search_key    = Column(UUID, default=lambda: str(uuid4()))
```

Comment:
If a group name happens to start with the EVENT_QR_ID, the search will always look for a QR code, won't it?